setup.py

```diff
@@ -0,0 +1,199 @@
+            type="win32"
+            name="Microsoft.Windows.Common-Controls"
+            version="6.0.0.0"
+            processorArchitecture="*"
+            publicKeyToken="6595b64144ccf1df"
+            language="*"
+        />
+    </dependentAssembly>
+  </dependency>
+</assembly>
+'''
+
+
+
+Wangview = Target(
+    # We can extend or override the VersionInfo of the base class:
+    # version = "1.0",
+    # file_description = "File Description",
+    # comments = "Some Comments",
+    # internal_name = "spam",
+
+    script="Wangview.py", # path of the main script
+
+    # Allows to specify the basename of the executable, if different from 'Wangview'
+    # dest_base = "Wangview",
+
+    # Icon resources:[(resource_id, path to .ico file), ...]
+    # icon_resources=[(1, r"Wangview.ico")]
+
+    other_resources = [(RT_MANIFEST, 1, (manifest_template % dict(prog="Wangview", level="asInvoker")).encode("utf-8")),
+    # for bitmap resources, the first 14 bytes must be skipped when reading the file:
+    #                    (RT_BITMAP, 1, open("bitmap.bmp", "rb").read()[14:]),
+                      ]
+    )
+
+
+# ``zipfile`` and ``bundle_files`` options explained:
+# ===================================================
+#
+# zipfile is the Python runtime library for your exe/dll-files; it
+# contains in a ziparchive the modules needed as compiled bytecode.
+#
+# If 'zipfile=None' is used, the runtime library is appended to the
+# exe/dll-files (which will then grow quite large), otherwise the
+# zipfile option should be set to a pathname relative to the exe/dll
+# files, and a library-file shared by all executables will be created.
+#
+# The py2exe runtime *can* use extension module by directly importing
+# the from a zip-archive - without the need to unpack them to the file
+# system.  The bundle_files option specifies where the extension modules,
+# the python dll itself, and other needed dlls are put.
+#
+# bundle_files == 3:
+#     Extension modules, the Python dll and other needed dlls are
+#     copied into the directory where the zipfile or the exe/dll files
+#     are created, and loaded in the normal way.
+#
+# bundle_files == 2:
+#     Extension modules are put into the library ziparchive and loaded
+#     from it directly.
+#     The Python dll and any other needed dlls are copied into the
+#     directory where the zipfile or the exe/dll files are created,
+#     and loaded in the normal way.
+#
+# bundle_files == 1:
+#     Extension modules and the Python dll are put into the zipfile or
+#     the exe/dll files, and everything is loaded without unpacking to
+#     the file system.  This does not work for some dlls, so use with
+#     caution.
+#
+# bundle_files == 0:
+#     Extension modules, the Python dll, and other needed dlls are put
+#     into the zipfile or the exe/dll files, and everything is loaded
+#     without unpacking to the file system.  This does not work for
+#     some dlls, so use with caution.

```

Comment:
They're automatically included in the setup script generated by py2exe' (which I've since edited). I think most of this information is on the py2exe website, but it's probably not doing any harm here. I guess they can be deleted, but if the script is ever regenerated, these comments will come back.